src/nemdeling/nemdeling.controller.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  Body,
+  Controller,
+  InternalServerErrorException,
+  Logger,
+  Post,
+  ServiceUnavailableException,
+} from "@nestjs/common";
+import { DisplayApiService } from "../display-api/display-api.service";
+import {
+  ApiBody,
+  ApiCreatedResponse,
+  ApiInternalServerErrorResponse,
+  ApiTags,
+} from "@nestjs/swagger";
+import { NemDelingService } from "./nemdeling.service";
+import { ServiceMessageBody, NemDelingResult, NemdelingSlide } from "./types";
+import { ApiServiceUnavailableResponse } from "@nestjs/swagger/dist/decorators/api-response.decorator";
+
+type RequestNormalizerResult = { [key: string]: NemdelingSlide[] };
+
+@ApiTags("NemDeling")
+@Controller("api/v1/nemdeling")
+export class NemDelingController {
+  constructor(
+    private readonly nemDelingService: NemDelingService,
+    private readonly displayApiService: DisplayApiService,
+    private readonly logger: Logger
+  ) {}
+
+  /**
+   * The template type to use when creating service message slides.
+   */
+  private readonly serviceMessageTemplateType = "Servicemeddelelse";
+
+  @Post("service-messages")
+  @ApiCreatedResponse({
+    description: "The service message was synced.",
+    type: String,
+  })
+  @ApiServiceUnavailableResponse({
+    description: "Application is busy processing.",
+  })
+  @ApiInternalServerErrorResponse({
+    description: "Application not able to process request.",
+  })
+  @ApiBody({ type: String })
+  async serviceMessage(@Body() body: ServiceMessageBody): Promise<string> {
+    const results: NemDelingResult[] = [];
+
+    if (
+      this.nemDelingService.serviceMessagesAreSyncing > 0 &&
+      this.nemDelingService.serviceMessagesAreSyncing < 5
```

Comment:
Why 5? Not that you need to change it, but a comment would help. Or, wrap the check in a function that you can then name something like shouldSync() or something like that.

Reply:
Just some number to ensure we never get stuck forever in the event of a crash